www/shared/data/index.js

```diff
@@ -0,0 +1,2 @@
+// Export Data
```

Comment:
I don't know if this was already discussed but I believe that the _data_ relative to a component should live next to that component. I think there are some use cases where `shared/data` must exist, but generally I believe we could enforce the usage of _data_ files next to its components.

# Proposal

Have files like

```js
.
├─ package.json
├─ package-lock.json
├─ ...
├─ www/
├── pages/
├──── home/
├───── index.js // Export the component here so we can import from 'pages/home'
├───── Home.js // Code of the component
├───── Home.css // Styles of the component
├───── Home.test.js // Test of the component
├───── Home.data.js // Data of the component or it's children if several use it
├─ ...
└─ README.md
```

This means that if some data is used in several places, then it should go up in the tree to the common place (e.g. if a page has two components with the same data, then it's the page that must have that data). If this is not possible then it should live in `shared/data`. With this we should also add rules to the JEST config to ignore _data_ files, so I suggest using those files with `.data.js` extension.